doc/gov/chapters/chapter09.md

```diff
@@ -81,3 +81,322 @@ Exceptions will be clearly recorded in Reference Architectures' Appendices which
 
 <a name="9.4"></a>
 ## 9.4 Adoption Roadmap
+
+<a name="9.5"></a>
+## 9.5 CNTT Field Trial/ Approach
+
+This portion of Chapter 9 is segmented into two subsections. Section 9.5.1 provides a summary and overview of the trials activities specifically targeted to potential trials participants. Section 9.5.2 addresses the overall approach of CNTT to field trials as a method of ensuring consistency between releases.
+
+<a name="9.5.1"></a>
+
+### 9.5.1 Summary/Field Trials Participants Overview
+
+CNTT has built a Reference Architecture (RA-1) on which Reference Implementation (RI-1) and Reference Conformance (RC-1) requirements have been defined. To ensure value add to Telco industry operators, suppliers, and end user customers, it is running field tests to validate the feasibility, utility, and effectiveness of its requirements and methods (RI-1/RC-1).
+
+<a name="9.5.1.1"></a>
+#### 9.5.1.1 Intents of Field Trials
+
+This is a partnership of CNTT with participants to validate the community is adopting a consistent approach. This is not a VI badging exercise. The trials will validate the RI-1 and the RC-1 test suite requirements and methods themselves, not VNFs or VI systems under test. 
+
+<a name="9.5.1.2"></a>
+#### 9.5.1.2 Expectations and Assumptions of Field Trials
+
+CNTT expects to exit the trial with either validation of RI-1 and RC-1 or a set of actions to close gaps. Through the community continuous improvement processes, a badging process that is mutually beneficial to operators and suppliers will be defined/refined. CNTT assumes that performance testing is not included in the field trial.
```

Comment:
Sentence issues, "Through the community continuous improvement processes, a ..."

Reply:
Updated made